pymc/sampling.py

```diff
@@ -2128,7 +2128,7 @@ def draw(
     vars
         A variable or a list of variables for which to draw samples.
     draws : int
```

Comment:
```suggestion
    draws : int, default 1
```

draws is optional, so it should have either optional or default. As here the default is of the documented type and a specific value, using default instead of optional is preferred, as explained in the third point of https://pymc-data-umbrella.xyz/en/latest/sprint/docstring_tutorial.html#parameters. The default would then be removed from the description below